sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs/EventProcessorClientAggregateEventsSample.java

```diff
@@ -129,10 +129,11 @@ public static void main(String[] args) throws Exception {
                     return client.send(batch);
                 }).block(operationTimeout);
             }
-        }).doFinally(signal -> {
-            logger.info("Disposing of producer.");
-            client.close();
-        }).subscribeOn(scheduler);
+        }).subscribeOn(scheduler)
```

Comment:
Does the order of `subscribeOn()` matter?

Reply:
According to this, order doesn't matter. 
https://stackoverflow.com/a/37986481/4220757